kong/clustering/init.lua

```diff
@@ -205,14 +205,34 @@ local function fill_empty_hashes(hashes)
   end
 end
 
-function _M:request_version_negotiation()
+
+function _M:call_control_plane(premature)
```

Comment:
I think the function could be just "version\_negotiation" (though the name is already used).